docsrc/admin/installation.md

```diff
@@ -1,19 +1,39 @@
 (installation)=
+
 # Installation
+
 This section provides instructions for installing the Data Library software on a server.
 
-Installation of the Data Library software is automated using [ansible](https://docs.ansible.com/ansible_community.html), a configuration management tool.
+Installation of the Data Library software is automated using [ansible](https://docs.ansible.com/ansible_community.html),
+a configuration management tool.
 
 ## Prepare the server
+
 Before running the installation script, perform the following steps to prepare the server:
-* Install CentOS 7 (minimal server configuration). For more information on this choice of operating system, see {ref}`infrastructure`.
-* Create a user account for the system administrator who will be performing the installation. Make that user a member of the `wheel` group so that they will be able to perform commands as root using `sudo`.
-* Mount a volume with at least 1TB of storage space, preferably with mirror RAID, at `/data`. List the volume in `/etc/fstab` to ensure that it will be mounted at boot time. We recommend using LVM to create logical volumes, and formatting the volume with XFS. Note that an XFS filesystem can be expanded but not shrunk, so it may be preferable to leave some disk space unallocated, to be used for snapshots or unanticipated storage needs, rather than putting all of the available space into the XFS-formatted volume.
+
+* Install CentOS 9 Stream from https://www.centos.org/centos-stream/#download. Select the appropriate architecture to
+  download the ISO version.
```

Comment:
If you're going to break long lines, please break them at 80 characters or less. This version of the source is harder to read in github than the original.

Reply:
OK, I'll do that.  It was just a pain for me to edit this on my laptop.  I realized after I reformatted I should have put the reformat in another commit.